src/components/Filter/FilterGroup.js

```diff
@@ -27,7 +35,8 @@ const FilterGroup = React.createClass({
 FilterGroup.propTypes = {
   groupName: PropTypes.string,
   tagItems: PropTypes.object,
-  onCheck: PropTypes.func
+  onCheck: PropTypes.func,
+  availableLessonsForTag: PropTypes.object
```

Comment:
i propTypes heter den `availableLessonsForTag`, men den både sendes inn som `availableLessons` og brukes lenger opp som `availableLessons`. Forøvrig, hvis den hadde hatt `.required` på seg, så ville du fått en warning om at den manglet...